homework_22102020/main.js

```diff
@@ -0,0 +1,90 @@
+// Bài 1
+function printTable(n) {
+    if (n < 1 || n > 10) return
+    console.log('In bảng nhân:')
+    for (let i = 1; i <= 10; i++) {
+        console.log(n + ' x ' + i + ' = ' + n * i)
```

Comment:
Dùng string template cho đỡ rối
Hơn nữa nhfin như đi copy của ai đó
```suggestion
        console.log(`${n} * ${i} = ${n * i}`)
```